http4k-contract/src/main/resources/org/http4k/contract/ui/public/oauth2-redirect.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en-US">
+<head>
+  <title>%%PAGE_TITLE%%</title>
+</head>
+<body>
+<script>
```

Comment:
I wish I could just load this script from the CDN (unsure about license implications and just maintainability in general), but even the CDN has the entire JS content embedded in the HTML.  I suppose the only other alternative would be to have the HttpHandler server the HTML directly from the CDN.